`yarn run test` causes test failures

```
$ yarn run test
yarn run v1.22.22
$ yarn workspaces run test

> @graphai/agent_filters
$ echo nothing
nothing

> @receptron/agentdoc
$ node --test --require ts-node/register ./tests/test_*.ts
receptron/graphai
✔ test getAgents (0.898551ms)
✔ test getPackageJson (0.364111ms)
✔ test getGitRep (0.788685ms)
✔ test getGitRep (0.157915ms)
✔ test getGitRep (0.123902ms)
✔ test getGitRep (0.099746ms)
✔ test getAgents (1.168617ms)
ℹ tests 7
ℹ suites 0
ℹ pass 7
ℹ fail 0
ℹ cancelled 0
ℹ skipped 0
ℹ todo 0
ℹ duration_ms 1152.901968

> @graphai/agents
$ node --test --require ts-node/register ./tests/**/test_*.ts
Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_any.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_any.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_any.ts (2202.459739ms)
  'test failed'

Error: Cannot find module './utils'
Require stack:
- /home/honma/git/graphai/llm_agents/anthropic_agent/lib/anthropic_agent.js
- /home/honma/git/graphai/llm_agents/anthropic_agent/lib/index.js
- /home/honma/git/graphai/agents/llm_agents/lib/index.js
- /home/honma/git/graphai/packages/agents/src/index.ts
- /home/honma/git/graphai/packages/agents/tests/graphai/test_dispatch.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/llm_agents/anthropic_agent/lib/anthropic_agent.js:9:17)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/llm_agents/anthropic_agent/lib/anthropic_agent.js',
    '/home/honma/git/graphai/llm_agents/anthropic_agent/lib/index.js',
    '/home/honma/git/graphai/agents/llm_agents/lib/index.js',
    '/home/honma/git/graphai/packages/agents/src/index.ts',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_dispatch.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_dispatch.ts (2235.088029ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_inputs.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_inputs.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_inputs.ts (2193.942903ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_loop.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_loop.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_loop.ts (1891.843698ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_multiple_functions.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_multiple_functions.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_multiple_functions.ts (1865.15395ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_nest.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_nest.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_nest.ts (2190.714293ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_params.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_params.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_params.ts (2229.565996ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_sample_flow.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_sample_flow.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_sample_flow.ts (2209.756532ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_simple.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_simple.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_simple.ts (1919.529179ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_simple_dispatch.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_simple_dispatch.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_simple_dispatch.ts (2255.124157ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_source_prop.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_source_prop.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_source_prop.ts (2233.334758ms)
  'test failed'

Error: Cannot find module './array_find_first_exists'
Require stack:
- /home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js
- /home/honma/git/graphai/agents/vanilla_agents/lib/index.js
- /home/honma/git/graphai/packages/test_utils/lib/test_runner.js
- /home/honma/git/graphai/packages/test_utils/lib/index.js
- /home/honma/git/graphai/packages/agents/tests/graphai/test_streamer.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1207:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/honma/git/graphai/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1038:27)
    at Module.require (node:internal/modules/cjs/loader:1289:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js:19:51)
    at Module._compile (node:internal/modules/cjs/loader:1521:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1623:10)
    at Object.require.extensions.<computed> [as .js] (/home/honma/git/graphai/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1266:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/honma/git/graphai/agents/vanilla_agents/lib/array_agents/index.js',
    '/home/honma/git/graphai/agents/vanilla_agents/lib/index.js',
    '/home/honma/git/graphai/packages/test_utils/lib/test_runner.js',
    '/home/honma/git/graphai/packages/test_utils/lib/index.js',
    '/home/honma/git/graphai/packages/agents/tests/graphai/test_streamer.ts'
  ]
}
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_streamer.ts (2252.179127ms)
  'test failed'

ℹ tests 12
ℹ suites 0
ℹ pass 0
ℹ fail 12
ℹ cancelled 0
ℹ skipped 0
ℹ todo 0
ℹ duration_ms 2283.439588

✖ failing tests:

test at tests/graphai/test_any.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_any.ts (2202.459739ms)
  'test failed'

test at tests/graphai/test_dispatch.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_dispatch.ts (2235.088029ms)
  'test failed'

test at tests/graphai/test_inputs.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_inputs.ts (2193.942903ms)
  'test failed'

test at tests/graphai/test_loop.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_loop.ts (1891.843698ms)
  'test failed'

test at tests/graphai/test_multiple_functions.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_multiple_functions.ts (1865.15395ms)
  'test failed'

test at tests/graphai/test_nest.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_nest.ts (2190.714293ms)
  'test failed'

test at tests/graphai/test_params.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_params.ts (2229.565996ms)
  'test failed'

test at tests/graphai/test_sample_flow.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_sample_flow.ts (2209.756532ms)
  'test failed'

test at tests/graphai/test_simple.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_simple.ts (1919.529179ms)
  'test failed'

test at tests/graphai/test_simple_dispatch.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_simple_dispatch.ts (2255.124157ms)
  'test failed'

test at tests/graphai/test_source_prop.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_source_prop.ts (2233.334758ms)
  'test failed'

test at tests/graphai/test_streamer.ts:1:1
✖ /home/honma/git/graphai/packages/agents/tests/graphai/test_streamer.ts (2252.179127ms)
  'test failed'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: /home/honma/.nvm/versions/node/v20.19.5/bin/node
Arguments: /home/honma/.nvm/versions/node/v20.19.5/lib/node_modules/yarn/lib/cli.js run test
Directory: /home/honma/git/graphai/packages/agents
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspaces for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/Collaboration.md
+++ b/Collaboration.md
@@ -24,6 +24,7 @@ Step 4. Run the test script
 Run the test
 
 ```sh
+yarn run build
 yarn run test
 ```
 

